fe/fe-core/src/main/java/org/apache/doris/external/jdbc/JdbcClient.java

```diff
@@ -306,6 +318,20 @@ public boolean isTableExist(String dbName, String tableName) {
         }
     }
 
+    public void getSpecifiedDatabaseMap(String specifiedDatabaseList) {
```

Comment:
Why is here another `getSpecifiedDatabaseMap()` method?
I think we unify this logic for all kinds of catalog

Reply:
it is better, let me fix it